twd.htm

```diff
@@ -219481,8 +219374,6 @@ <h3><a id="Year1997">1997</a></h3>
 5x Legal Manipulations
 13x Aire of Elation
 7x Majesty
-59 cards; 14 masters; 25 bleed actions; 13 bleed action mods; 7 combat
```

Comment:
I'd keep this as a comment on the deck construction - that all the actions are primarily intended as bleed seems a note worth preserving.